src/libs/actions/Session/index.js

```diff
@@ -316,20 +316,6 @@ function setPassword(password, validateCode, accountID) {
             // This request can fail if the password is not complex enough
             Onyx.merge(ONYXKEYS.ACCOUNT, {error: response.message});
         })
-        .catch((response) => {
-            if (response.title !== CONST.PASSWORD_PAGE.ERROR.VALIDATE_CODE_FAILED) {
-                return;
-            }
-
-            const login = lodashGet(response, 'data.email', null);
-            Onyx.merge(ONYXKEYS.ACCOUNT, {accountExists: true, validateCodeExpired: true, error: null});
-
-            // The login might not be set if the user hits a url in a new session. We set it here to ensure calls to resendValidationLink() will succeed.
```

Comment:
I actually don't really get what this comment is saying even though I asked for it to be added when a contributor added this 😅

Reply:
I think it cannot run... and could not run when the code was added...


Original state of file when this was added... 

https://github.com/akshayasalvi/App/blob/6180d9ecb90cbfb50562e047783a1813c5021ddd/src/libs/actions/Session/index.js#L215-L217